### PR TITLE
Add support for .env.development file

### PR DESCRIPTION
**Pull request by RaeesBhatti**
**- Summary**
Fixes: https://github.com/netlify/cli/issues/876

**- Test plan**
* Tests added

**- Description for the changelog**
* Add function `getEnvFile` to return `.env.development` or fallback to `.env` file
* Add tests for `getEnvFile`
* Pass project directory to detectors

**- A picture of a cute animal (not mandatory but encouraged)**
🦆

### CODE REVIEW

File: src/utils/env.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const { promisify } = require('util')
+
+const fileStat = promisify(fs.stat)
+
+async function getEnvFile(projectDir) {
+  const envDevelopmentFile = path.resolve(projectDir, '.env.development')
+  const envFile = path.resolve(projectDir, '.env')
+
+  try {
+    if ((await fileStat(envDevelopmentFile)).isFile()) return envDevelopmentFile
```
Comment (ehmicky) [inline src/utils/env.js:12]: That could be used too: https://github.com/sindresorhus/locate-path#usage

```js
return locatePath([envDevelopmentFile, envFile])
```

Comment (RaeesBhatti) [reply src/utils/env.js:12]: 👍🏽

**Comment by wesbos** [comment]
:D thank you for this!

**Comment by RaeesBhatti** [comment]
@wesbos Glad you found it useful 😊

**Pull request merged**